svf-llvm/lib/CHGBuilder.cpp

```diff
@@ -381,8 +381,14 @@ void CHGBuilder::analyzeVTables(const Module &M)
             SVFGlobalValue* pValue =
```

Comment:
Could we get the `GlobalObjVar` here directly?

Is this something you plan to do later?